### ISSUE TITLE
Phalangeal cone-shaped epiphyses

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7549]
**Preferred term label:**

Phalangeal cone-shaped epiphyses
**Synonyms**

**Definition (free text, please give PubMed ID)**
Cone-shaped epiphyses of the phalanges of the hands or feet. Cone-shaped epiphyses (also known as coned epiphyses) are epiphyses that invaginate into cupped metaphyses. That is, the epiphysis has a cone-shaped distal extension resulting from increased growth of the central portion of the epiphysis relative to its periphery.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
done
